src/main/java/chess/domain/location/Column.java

```diff
@@ -0,0 +1,70 @@
+package chess.domain.location;
+
+import chess.domain.board.Direction;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+public enum Column {
+    A(1),
+    B(2),
+    C(3),
+    D(4),
+    E(5),
+    F(6),
+    G(7),
+    H(8);
+
+    private static final Map<String, Column> CACHED_COLUMNS_BY_NAME = new HashMap<>();
+
+    static {
+        for (Column column : values()) {
+            CACHED_COLUMNS_BY_NAME.put(column.name(), column);
+        }
+    }
```

Comment:
```suggestion
        private static final Map<String, Column> CACHED_COLUMNS_BY_NAME = Arrays.stream(values())
            .collect(Collectors.toMap(Enum::name, Function.identity()));
```
좀 더 간단하게 표현할 수 있을것 같아요🙏

Reply:
와, 지금까지 `toMap()`을 사용하면 더 가독성이 떨어지던 경험이 많아서 자연스럽게 제외시켰던 것 같네요...!

바로 반영하겠습니다!